ee/ucp/kubernetes/layer-7-routing.md

```diff
@@ -19,87 +21,180 @@ Use an ingress controller when you want to:
 * Give your Kubernetes app an externally-reachable URL.
 * Load-balance traffic to your app.
 
-Kubernetes provides an NGINX ingress controller that you can use in Docker EE
-without modifications.
-Learn about [ingress in Kubernetes](https://v1-8.docs.kubernetes.io/docs/concepts/services-networking/ingress/).
+Ingress is an API object that manages external access to the services in a cluster, typically HTTP. In UCP 3.1, Ingress must be deployed separately. There are a variety of ingress controllers, but for UCP, the supported one is the [NGINX Ingress Controller](https://github.com/kubernetes/ingress-nginx). The following instructions provide highly-available production deployment of this controller. 
 
-## Create a dedicated namespace
+## Prerequisites
 
-1.  Navigate to the **Namespaces** page, and click **Create**.
-2.  In the **Object YAML** editor, append the following text.
-    ```yaml
-    metadata:
-      name: ingress-nginx
-    ```
+- UCP 3.1.X deployed and properly configured
```

Comment:
Not sure if we typically pin product version numbers within the text or somewhere else. This will be a bit tricky to maintain down the road.

Reply:
Good point - removing version #.